xrpl/models/requests/ledger.py

```diff
@@ -19,10 +17,8 @@ class Ledger(Request, LookupByLedgerRequest):
     """
 
     method: RequestMethod = field(default=RequestMethod.LEDGER, init=False)
-    accounts: bool = False
     transactions: bool = False
     expand: bool = False
     owner_funds: bool = False
     binary: bool = False
     queue: bool = False
```

Comment:
This change looks good to me.

But, the current model does not represent `ledger_hash`  or `ledger_index` fields in the `Ledger` request. I think we can include these changes in this PR (since there are no methods in this `Ledger` method)

Alternatively, I can note this down and work on it later too.

Reply:
I think I would like to we keep the scope of this PR to removing deprecated fields.  If we can make those changes in a separate PR that would be greatly appreciated 🙏

Reply:
> But, the current model does not represent `ledger_hash`  or `ledger_index` fields in the `Ledger` request.

It's included via inheritance - see `LookupByLedgerRequest`.